src/utils/commands/openWorkspace.ts

```diff
@@ -1,19 +1,16 @@
 import * as vscode from 'vscode';
-import {
-    AbstractCommand, CommandContext, PluginTreeItem, Resource
-} from '../../runtime';
 
+import { AbstractCommand } from '../../runtime';
 import { openWorkspaceFolder } from '../../utils/workspaceHelper';
 import { loggableAskError } from '../../exceptions';
 import { Logger } from '../../logger';
 
-
 export class OpenWorkspaceCommand extends AbstractCommand<void> {
     constructor() {
         super('ask.container.openWorkspace');
     }
 
-    async execute(context: CommandContext): Promise<void> {
+    async execute(): Promise<void> {
```

Comment:
Why was this removed? With the current eslint, I don't see any errors being shown on this even though it is not used in the method. Also, the abstract class method has this function variable as a required parameter. By deleting it here, how does typescript make sure that the abstract class methods are actually implemented, since the signature is different?

Reply:
Related to the lint rule I turned off above. Did some research: https://stackoverflow.com/questions/50266006/how-to-avoid-spurious-unused-parameter-warnings-in-typescript. But seems not a good workaround to follow ? We could discuss this one with other folks :)